Cargo.toml

```diff
@@ -15,8 +15,8 @@ itertools = "~0.9.0"
 thiserror = "1.0.23"
 
   [dependencies.bls]
-  package = "threshold_crypto"
-  version = "~0.4.0"
+  package = "blsttc"
+  version = "1.0.1"
```

Comment:
Use the tilde here as well, you'll want to use `~` deps for pretty much all library repos so that the app repos can easily get bug-fixes without making code changes to all it's dependencies